core-model/src/main/java/com/sequenceiq/cloudbreak/domain/FlowChainLog.java

```diff
@@ -7,9 +7,12 @@
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.NamedQuery;
 import javax.persistence.SequenceGenerator;
 
 @Entity
+@NamedQuery(name = "FlowChainLog.purgeOrphanFLowChainLogs",
```

Comment:
Sorry, I said named query, but recently we moved all of these queries to the repositories.

Reply:
nevermind, I'll move them then to repository